src/slice.cr

```diff
@@ -119,6 +119,38 @@ struct Slice(T)
     new(size, read_only: read_only) { value }
   end
 
+  # Creates a `Slice` to the given *pointer*, bounded by a null-terminator. This
+  # method does not allocate heap memory.
+  #
+  # The use of the optional `limit` is strongly recommended.
```

Comment:
*limit* argument here is **not** optional as far as I can see.

Reply:
I would agree. I just don't know if there are any use cases where it should be.

Reply:
I don't grok your drift... I meant to update the comment to the fact it is *required* for this `Pointer.new` overload.

Reply:
Ah, I ment for it to be applied to this method and the one below it. Which should have a :ditto:.

Reply:
I am just going to remove the option. It can be added later if its found to be needed.

Reply:
Argument itself I think is OK, it's just a matter of updating the docs.